.github/workflows/run-forge-tests.yaml

```diff
@@ -20,8 +25,11 @@ jobs:
       - name: Install dependencies
         run: forge install
 
+      - name: Install packages
+        run: yarn install --non-interactive --frozen-lockfile && yarn build
+
       - name: Run tests
-        run: forge test -vvv --via-ir
+        run: forge test -vvv --via-ir --ffi --fork-url https://mainnet.infura.io/v3/$INFURA_KEY
```

Comment:
I'm still learning the Forge APIs, but I think it might be possible to specify this forking at the test level, instead of having this be a global setting

Reply:
updated; we now fork during test setup instead